api/src/main/java/ca/bc/gov/educ/grad/report/api/service/GradReportService.java

```diff
@@ -265,6 +268,26 @@ public ResponseEntity<byte[]> getSchoolDistributionReport(ReportRequest reportRe
 		return response;
 	}
 
+	public ResponseEntity<byte[]> getSchoolLabelReport(ReportRequest reportRequest) {
+		String methodName = "getSchoolLabelReport(GenerateReportRequest reportRequest)";
```

Comment:
Note: ResponseEntity should be created at the controller layer

Reply:
I agree. But where have you been earlier? Now I have to rework all services, controllers and > 100 unit tests. Please create backlog ticket, I'll take care of it after completing my work with high priority tickets.

Reply:
Actually nothing wrong was there, because the class is part of API package, which is REST specific. It is aggregated service for all individual services, which are reusable outside of the web context.

Reply:
@arybakov-cgi relax. I am not asking you to move every existing ResponseEntity to the controller layer. I am just pointing out that the controller layer is responsible for processing request and response and the service layers handle all the business logic for future reference.

Reply:
@cditcher ups, late :). It is already done.